core/app/assets/javascripts/spree.js.erb

```diff
@@ -51,12 +51,35 @@ Spree.ajax = function(url, options) {
   return $.ajax(url, options);
 };
 
-Spree.routes = {
+/**
+ * @deprecated Spree.routes will be removed in a future release. Please use Spree.pathFor instead.
+ * See: https://github.com/solidusio/solidus/issues/3405
+ */
+const routes_deprecation_proxy = {
```

Comment:
I think this should be `Spree.routesDeprecationProxy` in order to be visible everywhere and to reflect JS naming conventions.

Reply:
Yes, you're absolutely right! Thanks!